import_test.R

```diff
@@ -0,0 +1,30 @@
+# install lib
+
+if (!suppressWarnings(require(rjson))) install.packages("rjson")
+if (!suppressWarnings(require(igraph))) install.packages("igraph")
+if (!suppressWarnings(require(dplyr))) install.packages("dplyr")
+if (!suppressWarnings(require(jsonlite))) install.packages("jsonlite")
+
+library("rjson")
+library("igraph")
+library("dplyr")
+library("jsonlite")
+
+
+
+#import json file from local machine
+
+json_file <- "~/data/network_analysis/trans_mod.json"
```

Comment:
```suggestion
json_file <- "trans_mod.json"
```
Better to use relative path and expect the user to set working directory, but I think this is a matter of taste.